src/LogFileReader.cpp

```diff
@@ -48,11 +53,35 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
+    QStringList splitLine = line.split(", ");
```

Comment:
splitLine could be given a better name to indicate that is the document was split into different sections